app/src/main/java/com/internship/move/feature/authentication/ForgotPasswordDialog.kt

```diff
@@ -0,0 +1,27 @@
+package com.internship.move.feature.authentication
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.DialogFragment
+import androidx.navigation.fragment.findNavController
+import com.internship.move.R
+import com.internship.move.databinding.DialogForgotPasswordBinding
+import com.zhuinden.fragmentviewbindingdelegatekt.viewBinding
+
+class ForgotPasswordDialog : DialogFragment(R.layout.dialog_forgot_password) {
+
+    private val binding by viewBinding(DialogForgotPasswordBinding::bind)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        dialog?.window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
```

Comment:
We try to avoid system colors because different OEMs might have different variations of them and it might make the app look different on Samsung device than a Pixel.
You can create a `transparent` color resource `#00000000` and call it here instead using `setBackgroundDrawableResource`